gettsim/parameters/soz_vers_beitr.yaml

```diff
@@ -164,19 +180,22 @@ soz_vers_beitr:
   2009-01-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.073
-      an: 0.082
+      allgemein: 0.146
+      zusatz: 0.009
+      ermäßigt: 0.140
     arbeitsl_v: 0.014
-  2010-01-01:
+  2009-07-01:
     deviation_from: previous
     ges_krankenv:
-      ag: 0.07
-      an: 0.079
+      allgemein: 0.14
+      ermäßigt: 0.134
```

Comment:
Parameter ermäßigt is never used in the code.

Do I remember correctly that we need it for #150 ?

Reply:
yes correct